### ISSUE TITLE
Make the prefix configurable

### ISSUE THREAD

**Issue by tmttn** [start-again/spookyBot#26]
## Feature Request

**Is your feature request related to a problem? Please describe.**
Making the prefix configurable allows the server admin to personalize it.
 
**Describe the solution you'd like**
Store the prefix in the database per server. If no prefix is configured, use the default one. Validate the prefix for the format `prefixName!`.

**Describe alternatives you've considered**
None 🤔 

**Teachability, Documentation, Adoption, Migration Strategy**
```
config: {
    command: 'setprefix',
    name: 'Set bot prefix',
    description: 'Change the default bot prefix',
    usage: `${prefix}setprefix <prefix>`,
    displayHelp: true,
    permissionNeeded: 'ADMINISTRATOR',
  },
```

**Comment by tmttn** [comment]
I would like to work on this ofc. ;-)

**Comment by LucasCtrl** [comment]
At the moment, I don't think this feature is necessary.
However, I'll leave it open if the necessity arises.

**Comment by tmttn** [comment]
I was trying out an implementation for this, but it became annoying very quickly, especially in the status / help messages... 😭

**Comment by ghost** [comment]
I don't think this is necessary, since there isn't any other bot with this prefix. Anyway, it's a good idea to implementate it. PRs welcome, and Lucas could do something too

**Comment by ghost** [comment]
I would do this with a JSON db, just like this:
{"serverId": "prefix" | "default prefix"}

**Comment by tmttn** [comment]
That's what I was doing, and that part is fine. It's mostly the status message that was giving me a headache, because that one is not server specific. It's specific to the bot user in general.
